bedrock/firefox/templates/firefox/developer-quantum-firstrun.html

```diff
@@ -0,0 +1,240 @@
+          </li>
+
+          <li class="gallery-item feature">
+            <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-memory.svg') }}" alt="">
+            <h3 class="name">{{ _('Memory') }}</h3>
+            <p class="desc">{{ _('Find memory leaks and make your application zippy.') }}</p>
+          </li>
+
+          <li class="gallery-item feature">
+            <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-web-audio.svg') }}" alt="">
+            <h3 class="name">{{ _('Web Audio') }}</h3>
+            <p class="desc">{{ _('The only developer tool for inspecting the Web Audio API.') }}</p>
+          </li>
+
+          <li class="gallery-item feature">
+            <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-scratchpad.svg') }}" alt="">
+            <h3 class="name">{{ _('Scratchpad') }}</h3>
+            <p class="desc">{{ _('Edit, write, run and execute JavaScript in real time.') }}</p>
+          </li>
+
+          <li class="gallery-item feature">
+            <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-style-editor.svg') }}" alt="">
+            <h3 class="name">{{ _('Style Editor') }}</h3>
+            <p class="desc">{{ _('Edit and manage all your CSS stylesheets in your browser.') }}</p>
+          </li>
+        </ul>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="section section-engage">
+    <div class="content">
+
+      <section class="speak-up">
+        <h3>{{ _('Speak up') }}</h3>
+        <p>{{ _('Feedback makes us better. Tell us how we can improve the browser and Developer tools.') }}</p>
+        <p class="cta"><a href="https://discourse.mozilla-community.org/c/devtools" class="button button-hollow">{{ _('Join the conversation') }}</a></p>
+      </section>
+
+      <section class="get-involved">
+        <h3>{{ _('Get involved') }}</h3>
+        <p>{{ _('Help build the last independent browser. Write code, fix bugs, make add-ons, and more.') }}</p>
+        <p class="cta"><a href="https://devtools-html.github.io/" class="button button-hollow">{{ _('Start now') }}</a></p>
+      </section>
+
+    </div>
+  </section>
+
+  <section class="section section-download">
+    <div class="content">
+      <div class="section-body">
+        <h2>{{ _('Want to be on the cutting-edge?') }}</h2>
+        <p>Firefox Nightly receives daily updates and allows you to access features months before they go mainstream.</p>
+        {{ download_firefox('nightly', platform='desktop', dom_id='footer-download', button_color='button-arrow', download_location='footer cta') }}
+      </div>
+    </div>
+  </section>
+</main>
+{% endblock %}
+
+{% block email_form %}{% endblock %}
+
+{# Bug 1381776 #}
+{% block update_notification %}{% endblock %}
+
+{% block js %}
+  {% javascript 'firefox-product-page' %}
```

Comment:
This doesn't seem to be the right JS bundle for this page (JS errors in the console from missing waypoints). Should the page have a secondary nav?

Reply:
Ah so, this was essentially copied from the product page. From the design, it does not require a secondary nav so, perhaps I do not need the JS at all. Will test locally and remove. Thanks for the heads up @alexgibson

Reply:
Yeah, no need for the JS if there's no subnav. I missed that on my first pass. Sorry!